microservice-session/pom.xml

```diff
@@ -151,24 +157,35 @@
             </build>
         </profile>
     </profiles>
+    <dependencyManagement>
```

Comment:
this section is not required because you define quarkus-bom in the parent pom.xml

Reply:
Can you please check now I have changed required changes